app/assets/stylesheets/application.css.sass

```diff
@@ -1,5 +1,2 @@
-/* *= require_tree .
-/* *= require_self
```

Comment:
In the rails instructions for the`bootstrap-sass` [Readme](https://github.com/twbs/bootstrap-sass), it says to git rid of these (^^^) in favor of the `@import` syntax, but maybe I'd want them in there if I'm trying to use a custom stylesheet as well?